Packs/Zoom/Integrations/Zoom/Zoom_test.py

```diff
@@ -935,3 +967,676 @@ def test_zoom_fetch_recording_command__not_able_to_delete(mocker):
+    """
+        Given -
+           client
+        When -
+            asking for a list public channels
+        Then -
+            Validate that a func has been called
+    """
+    client = Client(base_url='https://test.com', account_id="mockaccount", client_id="mockclient", client_secret="mocksecret")
+    channel_id = "channel_id"
+    user_id = "user_id"
+
+    expected_url_suffix = f'users/{user_id}/channels/{channel_id}' if channel_id else f'users/{user_id}/channels'
+
+    expected_raw_data = {
+        "channels": [
+            {"jid": "channel_jid_1_t", "id": "channel_id_1", "name": "Channel 1", "type": "public",
+                "channel_url": "https://test1.com", "next_page_token": "token1"},
+            {"jid": "channel_jid_2", "id": "channel_id_2", "name": "Channel 2", "type": "public",
+                "channel_url": "https://test1.com", "next_page_token": "token2"}
+        ]
+    }
+
+    expected_results = {
+        'UserChannelsNextToken': None,
+        "channels": [
+            {"jid": "channel_jid_1_t", "id": "channel_id_1", "name": "Channel 1", "type": "public",
+                    "channel_url": "https://test1.com", "next_page_token": "token1"},
+            {"jid": "channel_jid_2", "id": "channel_id_2", "name": "Channel 2", "type": "public",
+                    "channel_url": "https://test1.com", "next_page_token": "token2"}
+        ]
+    }
+
+    zoom_list_user_channels_mock = mocker.patch.object(client, "zoom_list_user_channels")
+    zoom_list_user_channels_mock.return_value = expected_raw_data
+
+    from Zoom import zoom_list_user_channels_command
+
+    result = zoom_list_user_channels_command(
+        client,
+        channel_id=channel_id,
+        user_id=user_id
+    )
+
+    zoom_list_user_channels_mock.assert_called_with(
+        user_id=user_id,
+        page_size=50,
+        next_page_token=None,
+        url_suffix=expected_url_suffix,
+        page_number=1
+    )
+
+    assert result.outputs == expected_results
+    assert result.outputs['channels'][0]['id'] == expected_results['channels'][0]['id']
+    assert result.outputs['channels'][0]['jid'] == expected_results['channels'][0]['jid']
+
+
+def test_zoom_list_user_channels_command__limit(mocker):
+    """
+        Given -
+           client
+        When -
+            asking for a limit of results
+        Then -
+            Validate that a func that runs a pagination has been called
+    """
+    manual_list_user_channel_pagination_mock = mocker.patch.object(Zoom, "manual_list_user_channel_pagination")
+    mocker.patch.object(Client, "generate_oauth_token")
+    client = Client(base_url='https://test.com', account_id="mockaccount",
+                    client_id="mockclient", client_secret="mocksecret")
+    from Zoom import zoom_list_user_channels_command
+    zoom_list_user_channels_command(client=client, user_id="user_id", limit=5)
+    assert manual_list_user_channel_pagination_mock.called
```

Comment:
general comment also for other tests, if you check that command was called, validate also the arguments values.